src/summary/LongitudinalTable.jsx

```diff
@@ -87,6 +86,20 @@ export default class LongitudinalTable extends Component {
         }
         return <div />;
     }
+    renderNameCell(name) {
+        return (
+            <Tooltip
+                placement='right'
+                overlayClassName={`option-tooltip`}
```

Comment:
Did we just copy this class over from another file? We should come up with our own class here so we can intentionally style the width of the cells so they're not extremely large when unnecessary. IMO.